AzureTable/Microsoft.DataTransfer.AzureTable/Source/AzureTableSourceAdapter.cs

```diff
@@ -16,6 +16,7 @@ sealed class AzureTableSourceAdapter : IDataSourceAdapter
         private readonly IAzureTableSourceAdapterInstanceConfiguration configuration;
         private readonly CloudTable table;
         private readonly TableQuery query;
+        private readonly TableRequestOptions tRequestOptions;
```

Comment:
`t` at the front does not make too much sense. Use either `tableRequestOptions`, or `requestOptions`, as this entire thing only deals with tables anyway.